### PR TITLE
Migrate 'cmd' to 'main', licence update

### LINKED ISSUE #94: Move cmd package into main
Does it really need to be in a separate package? It's already at the top of the package food chain and the only thing `main` does is... call a function in `cmd`

My bad, tbt #46

### LINKED ISSUE #98: Standardise licences
Right now we have two licences going on:
- `Apache 2.0` in individual Go files
- `MIT` in our repository licence

paging @chadlagore

### PR DESCRIPTION
**Pull request by bobheadxi**
:tickets: **Ticket(s)**: 
- Closes #94 
- Closes #98 

---

## :construction_worker: Changes

This is largely a simple code migration. ⏫ 

* All command-line functionality now in 'main' package
* 'inertia daemon' commands are now completely disabled outside of the inertia docker image to prevent someone from inadvertently starting up a daemon locally or, even worse, name one of their remotes `daemon` and having it conflict with the command 😢 
* Removed all references to the `Apache 2.0` licence - we are already under `MIT`

## :flashlight: Testing Instructions

```bash
go install
inertia daemon run # should no longer work
```

### CODE REVIEW

File: client/bootstrap/daemon-up.sh [hunk 1]
```diff
@@ -33,7 +33,7 @@ sudo docker run -d --rm \
     -p "$PORT":8081 \
     -v /var/run/docker.sock:/var/run/docker.sock \
     -v $HOME:/app/host \
-    -e DAEMON="true" \
+    -e INERTIA_DAEMON='true' \
```
Comment (PiggySpeed) [inline client/bootstrap/daemon-up.sh]: Just an observation - it looks like use of " " and ' ' is inconsistent in our shell scripts.

Comment (bobheadxi) [reply client/bootstrap/daemon-up.sh]: That's a good point... I'm not actually sure how interchangeable they are 🤔 

Will look into it

**Comment by bobheadxi** [comment]
Thanks for the review @PiggySpeed!

**Comment by bobheadxi** [comment]
@rogermyang This one will affect your ticket (#72), remember to make sure you are up to date with `master` when I merge later 😄

File: client/bootstrap/daemon-up.sh (hunk 1 above)
Comment (bobheadxi) [reply client/bootstrap/daemon-up.sh]: I think the way we have it now is fine: [see this explanation for single vs double quotes in bash](https://stackoverflow.com/questions/6697753/difference-between-single-and-double-quotes-in-bash)

Single for string literals, double for things like `"$MYVAR"`

Comment (PiggySpeed) [reply client/bootstrap/daemon-up.sh]: Cool, thanks for the link!

**Pull request merged**